stdlib/3/inspect.pyi

```diff
@@ -41,4 +161,129 @@ FullArgSpec = NamedTuple('FullArgSpec', [('args', List[str]),
+                  formatvarargs: Optional[Callable[[str], str]] = ...,
+                  formatvarkw: Optional[Callable[[str], str]] = ...,
+                  formatvalue: Optional[Callable[[Any], str]] = ...,
+                  formatreturns: Optional[Callable[[Any], str]] = ...,
+                  formatannotations: Optional[Callable[[Any], str]] = ...,
+                  ) -> str: ...
+def formatargvalues(args: List[str],
+                    varargs: Optional[str] = ...,
+                    varkw: Optional[str] = ...,
+                    locals: Optional[Dict[str, Any]] = ...,
+                    formatarg: Optional[Callable[[str], str]] = ...,
+                    formatvarargs: Optional[Callable[[str], str]] = ...,
+                    formatvarkw: Optional[Callable[[str], str]] = ...,
+                    formatvalue: Optional[Callable[[Any], str]] = ...,
+                    ) -> str: ...
+def getmro(cls: type) -> Tuple[type, ...]: ...
+
+# Python 3.2+
+def getcallargs(func: Callable[..., Any],
+                *args: Any,
+                **kwds: Any) -> Dict[str, Any]: ...
+
+
+# Python 3.3+
+ClosureVars = NamedTuple('ClosureVars', [('nonlocals', Mapping[str, Any]),
+                                         ('globals', Mapping[str, Any]),
+                                         ('builtins', Mapping[str, Any]),
+                                         ('unbound', AbstractSet[str]),
+                                         ])
+def getclosurevars(func: Callable[..., Any]) -> ClosureVars: ...
+
+# Python 3.4+
+def unwrap(func: Callable[..., Any],
+           *,
+           stop: Callable[[Any], Any]) -> Any: ...
+
+
+#
+# The interpreter stack
+#
+
+# Python 3.5+ (functions returning it used to return regular tuples)
+FrameInfo = NamedTuple('FrameInfo', [('frame', FrameType),
+                                     ('filename', str),
+                                     ('lineno', int),
+                                     ('function', str),
+                                     ('code_context', List[str]),
+                                     ('index', int),
+                                     ])
+
+# TODO make the frame type more specific
+def getframeinfo(frame: Any, context: int = 1) -> FrameInfo: ...
+def getouterframes(frame: Any, context: int = 1) -> List[FrameInfo]: ...
+def getinnerframes(traceback: TracebackType, context: int = 1) -> List[FrameInfo]:
+    ...
+def currentframe() -> Optional[FrameType]: ...
+def stack(context: int = 1) -> List[FrameInfo]: ...
+def trace(context: int = 1) -> List[FrameInfo]: ...
+
+#
+# Fetching attributes statically
+#
+
+# Python 3.2+
+def getattr_static(obj: object, attr: str, default: Optional[Any] = ...) -> Any: ...
+
+
+#
+# Current State of Generators and Coroutines
+#
+
+# TODO In the next two blocks of code, can we be more specific regarding the
+# type of the "enums"?
+
+# Python 3.2+
+GEN_CREATED = ...  # type: Any
```

Comment:
I think these could be declared as `str` -- making them `Any` shuts up all type checking related to them completely.